### ISSUE TITLE
Variable "CraftClient" does not exist.

### ISSUE THREAD

**Issue by bartrylant** [verbb/feed-me#293]
### Description
After selecting the primary element in an XML file, if I click "save & continue" (to go to the field mapping page) I get this error:

```
Twig Runtime Error – Twig_Error_Runtime
Variable "CraftClient" does not exist.
in /srv/users/serverpilot/apps/***-app/vendor/verbb/feed-me/src/templates/_includes/elements/entry/map.html
```

### Additional info
- Plugin version: 3.0.0-beta.2
- Craft version: Craft Pro 3.0.1

**Comment by Nordicdesign** [comment]
Same issue here at the same point, I've attached the stack trace. 
I'm trying to import a wordpress generated XML file on an empty local/MAMP version of craft.

Craft 3.0.1 Solo
Feed Me 3.0.0-beta.2

[stack-trace.txt](https://github.com/verbb/feed-me/files/1891609/stack-trace.txt)

**Comment by engram-design** [comment]
This should be fixed in [3.0.0-beta.3](https://github.com/verbb/feed-me/releases/tag/3.0.0-beta.3)
